### PR TITLE
Replace dependabot with GitHub Action

### PR DESCRIPTION
**Pull request by quietbits**
This will replace multiple issues from `dependabot` for a few individual dependencies with a single issue to update all of them (which is our practice).

### CODE REVIEW

File: .github/workflows/update_deps.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  create_issue:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Update dependencies
+        uses: imjohnbo/issue-bot@v3.3.4
```
Comment (marcelosalloum) [inline .github/workflows/update_deps.yml:15]: GH recommends pinning the actions to their commit sha instead of the tag, since the sha is immutable while the tag can be changed to point to a different commit.

```suggestion
        uses: imjohnbo/issue-bot@7e438653c0da13f4f79678a56bb9ecbd9dcc26ac
```

Comment (leighmcculloch) [inline .github/workflows/update_deps.yml:15]: ❗ GitHub had a list of things they recommend we do whenever using third party actions:

- Pin actions to a full length commit SHA

- Audit the source code of the action

- Pin actions to a tag only if you trust the creator

Ref: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Can you do the first two items?

File: .github/workflows/update_deps.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          title: "Update dependencies"
+          body: |
+            Update all project dependencies
+            ```
+            yarn upgrade-interactive --latest
+            ```
```
Comment (leighmcculloch) [inline .github/workflows/update_deps.yml:22]: 💡 Do you have a checklist of things you do for the upgrade that we could knowledge share here? Also, it might be helpful to include commands for how you look at diffs of dependencies being upgraded, so everyones process converges for how to do this.

**Review by leighmcculloch — COMMENTED** [review]
One request (❗), and one suggestion (💡).

**Review by marcelosalloum — APPROVED** [review]
LGTM! I've just added one suggestion to pin the GH Action commit sha instead of its tag. [GH Security](https://github.com/imjohnbo/issue-bot/commit/7e438653c0da13f4f79678a56bb9ecbd9dcc26ac) recommends this:
> Pin actions to a full length commit SHA
